pkgs/development/python-modules/asf_search/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytz, shapely, importlib-metadata, requests, python-dateutil }:
+
+buildPythonPackage rec {
+  pname = "asf_search";
+  version = "3.0.6";
+
+  src = fetchFromGitHub {
+    owner = "asfadmin";
+    repo = "Discovery-${pname}";
+    rev = "v${version}";
+    sha256 = "1jzah2l1db1p2mv59w9qf0x3a9hk6s5rzy0jnp2smsddvyxfwcyn";
+  };
+
+  doCheck = false;
```

Comment:
Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.